feature3.java

```diff
@@ -0,0 +1,6 @@
+import java.io.*;
+class Solution{
```

Comment:
please rename this to `Feature3`, solution is not a good class name.

Reply:
done with the changes.